Mathlib/Analysis/NormedSpace/ENormedSpace.lean

```diff
@@ -34,7 +34,6 @@ normed space, extended norm
 
 noncomputable section
 
-attribute [local instance] Classical.propDecidable
```

Comment:
This one was causing instance diamonds

Reply:
Please explain in the commit message which instance used classical because of this.

Reply:
> Please explain in the commit message which instance used classical because of this.

Otherwise LGTM. Thanks!

bors d+

Reply:
I was wrong about this, but I've updated the PR description to explain why this is such an antipattern.